src/Entity/Embargo.php

```diff
@@ -42,7 +42,8 @@
  *       "html" = "Drupal\Core\Entity\Routing\AdminHtmlRouteProvider"
  *     },
  *   },
- *   list_cache_tags = { "node_list", "media_list", "file_list" },
```

Comment:
As per [Drupal's `EntityTypeInterface::getListCacheTags()`](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21EntityTypeInterface.php/interface/EntityTypeInterface/10), which should be:

> The list cache tags associated with this entity type.
>
> Enables code listing entities of this type to ensure that newly created entities show up immediately.

Really could leave unset, as the default is `embargo_list`, but whatever, we can set explicitly.

Thinking this was originally set as it was in a misguided attempt to have the _effects_ of the embargo applied immediately, but this fits more inline with `::getListCacheTagsToInvalidate()`, which is used when invalidating cache tags on entity save/delete, instead of providing the tags/contexts to set where listing of the given embargo entities are used, such as when calculating the access control results.